x-pack/plugins/actions/README.md

```diff
@@ -446,15 +446,18 @@ The PagerDuty action uses the [V2 Events API](https://v2.developer.pagerduty.com
+| severity      | The perceived severity of on the affected system. This can be one of `critical`, `error`, `warning` or `info`_(default)_.                                                                                                                                                                                               | string _(optional)_             |
+| timestamp     | An [ISO-8601 format date-time](https://v2.developer.pagerduty.com/v2/docs/types#datetime), indicating the time the event was detected or generated.                                                                                                                                                                     | string _(optional)_            |
+| component     | The component of the source machine that is responsible for the event, for example `mysql` or `eth0`.                                                                                                                                                                                                                   | string _(optional)_             |
+| group         | Logical grouping of components of a service, for example `app-stack`.                                                                                                                                                                                                                                                   | string _(optional)_             |
+| class         | The class/type of the event, for example `ping failure` or `cpu load`.                                                                                                                                                                                                                                                  | string _(optional)_             |
+| links         | An array of links to attach to the event                                                                                                                                                                                                                                                                                | {href,text}[] _(optional)_      |
+| images        | An array of images to attach to the event                                                                                                                                                                                                                                                                               | {src,href,alt}[] _(optional)_   |
+| customDetails | A map of custom properties to attach to the event                                                                                                                                                                                                                                                                       | Map[string,string] _(optional)_ |
```

Comment:
We're working on this but we currently have duplication between this file and `docs/user/alerting/action-types/pagerduty.asciidoc` for now. Would you be able to copy these lines to the user docs as well?

I'll make sure we have an issue to cleanup the README.md file so it doesn't repeat what's in the docs.